tests/lib/rules/jsx-indent.js

```diff
@@ -212,17 +214,6 @@ ruleTester.run('jsx-indent', rule, {
     ].join('\n'),
     parserOptions: parserOptions
   }, {
-    // Literals indentation is not touched
```

Comment:
I think that Literal indentation should be touched as it doesn't make a difference in the resulting HTML (different for JSXExpressions, but that doesn't apply here).